Digipost.Signature.Api.Client.Core/JobCustomizations.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Digipost.Signature.Api.Client.Core.Enums;
+
+namespace Digipost.Signature.Api.Client.Core
+{
+    public interface IJobCustomizations
```

Comment:
Kunne dette like gjerne vært lagt inn i [`ISignatureJob`](https://github.com/digipost/signature-api-client-dotnet/blob/dev/Digipost.Signature.Api.Client.Core/ISignatureJob.cs) i stedet?

Reply:
Absolutt!

Reply:
Var litt raskesen og alt for gira der. 😁